app/logic/Mail/EMail.ts

```diff
@@ -84,6 +85,8 @@ export class EMail extends Message {
   storageLock = new Lock();
   /** For composer only. Optional. */
   identity: MailIdentity;
+  /* Only used when constructing iMIP outgoing messages */
+  method: iCalMethod | undefined;
```

Comment:
`method` is too generic, if on email.
maybe `iCalMethod`?

Reply:
Done in 812d19a